static/js/translations.js

```diff
@@ -202,4 +210,425 @@ translations.es = {
+    supporters_2: 'Supporters? Segments de marché?',
+    foes_1: 'Les personnes les plus opposées au changement sont…',
+    foes_2: 'Ennemis?',
+    ideate: 'Conception',
+    ux_1: 'La façon dont cela fonctionnera est…',
+    ux_2: 'Quelle est l\'expérience utilisateur?',
+    changes_1: 'Pour cela se produise, ces changement doivent se produire…',
+    changes_2: 'Qui doit faire quoi pour cela se produise?',
+    partners_1: 'Pour mettre en oeuvre cette idée, je pourrai collaborer avec…',
+    partners_2: 'Partenaires clés pour la mise en place? Partenaires clés pour aider d\'autres à évaluer votre proposition de valeur?',
+    events_1: 'La raison pour agir maintenant est…',
+    events_2: 'Quels sont les événements déclencheurs?',
+    field_1: 'Autres acteurs',
+    field_2: 'Qui d\'autre est dans ce secteur?',
+    approach_1: 'Mon approche est meilleure parce que…',
+    approach_2: 'Qu\'est-ce que qui ne va pas? Qu\'est qui manque? Qu\'est-ce qui ne marche pas?',
+    resources_1: 'Ressources clés (outre financières) pour réussir',
+    resources_2: 'Ressources physiques, humaines et intellectuelles?',
+    prototype_test: 'Prototype et Test',
+    activities_1: 'Ma stratégie pour le déploiement de ce projet implique les activités clés suivantes',
+    activities_2: 'Prochaines étapes? Projets pilotes?',
+    costs_1: 'Je vais soutenir ce projet en',
+    costs_2: 'Structure de coût? Viabilité financière? Sources de revenu?',
+    risk_1: 'Mon risque principale est…',
+    risk_2: 'Qu\'est-ce qui peut tourner mal? Comment le problème pourrait évoluer? Quels sont les obstacles juridiques, culturels et autres',
+    adoption_1: 'Je vais sensibiliser au sujet de mon projet en…',
+    adoption_2: 'Comment vais-je promouvoir l\'adoption?',
+    footer_1: 'Sponsorisé par',
+    footer_2: 'Note',
+    footer_3: 'Le Public Project Canvas est une initiative de Beth Simone Noveck pour le GovLab et est distribué sous licence',
+    footer_4: 'Creative Commons Attribution-ShareAlike 4.0 International License.',
+    modal_1: 'Editer le mot de passe du Canvas',
+    modal_2: 'Ancien mot de passe (si il existe)',
+    modal_3: 'Nouveau mot de passe (laisser vide pour un accès public)',
+    modal_4: 'Confirmez le mot de passe (laisser vide pour un accès public)',
+    modal_5: 'Sauvegarder',
+    modal_6: 'Mot de passe',
+    modal_7: 'Soumettre',
+    notify_1: 'Mot de passe incorrect',
+    notify_2: 'Ce Canvas existe déjà',
+    notify_3: 'Les mots de passe ne coïncident pas',
+    notify_4: 'Canvas sauvegardé',
+    notify_5: 'Erreur en changeant de mot de passe',
+};
+
+translations.de = {
+    language_0: 'de',
+    language_1: 'English',
+    language_2: 'Spanish',
+    language_3: 'Dutch',
+    language_4: 'French',
+    language_5: 'German',
+    language_6: 'Italian',
+    canvas: 'Leinwand',
+    canvas_saved: 'Gespeicherte Leinwände',
+    instructions_1: 'Die GovLab Akademie Leinwand',
```

Comment:
Yeah I think we should generally refrain from translating the equivalent of brand names